policyengine_us/parameters/gov/states/oh/tax/income/credits/adoption/age_threshold.yaml

```diff
@@ -0,0 +1,15 @@
+description: Ohio qualifies children for the adoption credit below this age threshold.
+values:
+  2021-01-01: 18
+  2023-01-01: 0
```

Comment:
Instead of this, please add the credit to the parameter listing either refundable or non-refundable credits for 2021-2022, then remove it from 2023